### PR TITLE
Fix console

### PR DESCRIPTION
**Pull request by 3y3**

### CODE REVIEW

File: lib/InjectorServer.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ function injectorServer(options) {
     // Get previously cached mirror if existed
     mirror = resolveCachedMirror(cache, mirror);
 
-    var serializer = makeMirrorSerializer(true);
+    var serializer = makeMirrorSerializer(true, {
+      maxStringLength: 10000
```
Comment (3y3) [inline lib/InjectorServer.js]: We store this constant in two places at current time. (this is bad solution and I can rewrite it on something more complex, if you want)

**Comment by 3y3** [comment]
@bajtos , ready to review.

File: lib/InjectorServer.js (hunk 1 above)
Comment (bajtos) [inline lib/InjectorServer.js]: Perhaps we should use `-1` instead? AFAIK that way V8 does not truncate the response values at all.

**Comment by bajtos** [comment]
@3y3 I am afraid I don't have enough time to review this in detail. I don't see any obvious problem, if the patch works for you then I am fine with it. :shipit:

**Comment by 3y3** [comment]
> Perhaps we should use -1 instead

Everywhere? Or only for console? I'm :+1:  for `-1`

**Comment by bajtos** [comment]
> Everywhere? Or only for console? I'm :+1: for -1

I don't have a strong opinion. Shall we try to use it everywhere and see if it breaks anything?

**Pull request merged**